src/pages/Home/Home.page.js

```diff
@@ -1,10 +1,51 @@
 import React from "react";
-import universalStyles from "./home.module.scss"
+import "./homePage.scss";
+import imgIntro from "./imgHome/img-intro.png";
+import dec from "./imgHome/dec.svg"
+import bicicleta1 from "./imgHome/bicicleta1.png"
+import bicicleta2 from "./imgHome/bicicleta2.png"
+import bicicleta3 from "./imgHome/bicicleta3.png"
+
 const HomePage = () => {
     return (
-        <div className={universalStyles["container"]}>
-            iuhfgvcxiuhgduifcv
+        <div className="home-container">
+
+            <div className="header-post">
+
+                <img className="header-post-dec" src={dec} alt="dec" />
+                <div className="header-post-text">
+
+                    <div className="header-post-title">bicicletas feitas sob medida<span className="orange-point">.</span></div>
+                    <div className="header-post-description">Bicicletas elétricas de alta precisão e qualidade,  feitas sob medida para o cliente. Explore o mundo na sua velocidade com a Bikcraft.</div>
+                    <button className="header-post-button">Selecione a sua</button>
+                </div>
+                <img src={imgIntro} alt="img-intro" />
+
+            </div>
+
+            <div className="catalog-container">
+                <div className="catalog-title">escohola a sua<span className="orange-point">.</span></div>
+                <div className="catalog-cards">
+                    <div className="catalog-card-item">
+                        <img className="catalog-card-img" src={bicicleta2 } alt="" />
+                        <div className="catalog-card-description"><li className="orange-mark"></li>Magic Migth</div>
```

Comment:
```suggestion
                        <div className="catalog-card-description">Magic Migth</div>
```

use pseudo-element for mark

Reply:
done